src/vs/workbench/contrib/multiDiffEditor/browser/multiDiffEditor.ts

```diff
@@ -81,6 +86,23 @@ export class MultiDiffEditor extends AbstractEditorWithViewState<IMultiDiffEdito
 		if (viewState) {
 			this._multiDiffEditorWidget!.setViewState(viewState);
 		}
+		this._reveal(options);
+	}
+
+	override setOptions(options: IEditorOptions | undefined): void {
+		this._reveal(options);
+	}
+
+	private _reveal(options: IEditorOptions | undefined): void {
+		const viewState = options?.viewState;
+		if (!viewState || !('revealData' in viewState)) {
+			return;
+		}
+		const revealData: any = viewState.revealData;
+		if ('range' in revealData && Range.isIRange(revealData.range)
```

Comment:
Use an interface here and cast `viewState` to that interface.
You can assume that the shape is good, no need to check it.